nextflow/rmatsTurbo.nf

```diff
@@ -0,0 +1,232 @@
+nextflow.enable.dsl=2
+
+
+process rmats_prep_g1 {
```

Comment:
Can there just be a single `rmats_prep` process? The code for `rmats_prep_g2` is mostly identical

Reply:
I have the same question too. If I want to put every thing in one nextflow file, I have to write two `process` for the same task.

The modern way to solve this is to make the workflow modularized in separate files, then `include` modularized `process`es in the `main workflow` and use the alias like the WDL does. 

To put all code in one file, this is the way to do.

Reply:
Okay, I put the rmats code in the module file.